context/context.go

```diff
@@ -55,10 +55,10 @@ type NSSFContext struct {
 	RegisterIPv4 string
 	// HttpIpv6Address string
```

Comment:
What about deleting this line? If so, it can be done in another PR. What do you think?
```suggestion
```